### ISSUE TITLE
Missing headers in umbrella header

### ISSUE THREAD

**Issue by mmackowiak** [jedisct1/swift-sodium#35]
Hi,

we're using swift-soidum as a Carthage module - it's working flawlessly, however, in most of cases project compilation returns warnings:
```
<module-includes>:1:1: warning: umbrella header for module 'Sodium' does not include header 'crypto_int32.h'
#import "Headers/Sodium.h"
^
<module-includes>:1:1: warning: umbrella header for module 'Sodium' does not include header 'crypto_int64.h'
#import "Headers/Sodium.h"
^
<module-includes>:1:1: warning: umbrella header for module 'Sodium' does not include header 'crypto_sign_edwards25519sha512batch.h'
#import "Headers/Sodium.h"
^
<module-includes>:1:1: warning: umbrella header for module 'Sodium' does not include header 'crypto_uint16.h'
#import "Headers/Sodium.h"
^
<module-includes>:1:1: warning: umbrella header for module 'Sodium' does not include header 'crypto_uint32.h'
#import "Headers/Sodium.h"
^
<module-includes>:1:1: warning: umbrella header for module 'Sodium' does not include header 'crypto_uint64.h'
#import "Headers/Sodium.h"
^
<module-includes>:1:1: warning: umbrella header for module 'Sodium' does not include header 'crypto_uint8.h'
#import "Headers/Sodium.h"
^
```
It seems, that these headers are not even used - should they be added to `sodium_lib.h` or simply removed?

Thanks!

**Comment by blochberger** [comment]
This should be already fixed by #25.

**Comment by mmackowiak** [comment]
Oh, thank you very much. We stick to `v0.1` since `v0.2` fails to be built by Carthage, but I will have a closer look on it.
